### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by rachfern**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
- Analyze data in CSVs. Inspect data for any data quality issues

## What did you learn from the changes you have made?
- Looking at the mean, min, and max of datasets

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
- No

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
- None faced this time

## How were these changes tested?
- Tested in VS Code

## A reference to a related issue in your repository (if applicable)
- N/A

## Checklist
- [X ] I can confirm that my changes are working as intended

### CODE REVIEW

File: 02_activities/assignments/assignment_2.ipynb
Comment (anjali-deshpande-hub) [inline 02_activities/assignments/assignment_2.ipynb]: The call to check_zeros is incorrect

**Review by anjali-deshpande-hub — CHANGES_REQUESTED** [review]
The call to check_zeros function is incorrect. Please correct and resubmit.

File: 02_activities/assignments/assignment_2.ipynb
Comment (rachfern) [reply 02_activities/assignments/assignment_2.ipynb]: I have resubmitted it. Please let me know if it is correct now. Thank you!

**Review by anjali-deshpande-hub — CHANGES_REQUESTED** [review]
Looks good. 
Another way to do the same thing
```
if check_zeros(patient_means):
      return "True"
  else:
      return "False"
```
can be

`return check_zeros(patient_mean)`

There is another thing I overlooked in my first review. This PR also has changes to assignment_1.ipynb. It should only have changes to assignment_2.ipynb. Please correct that.

File: 02_activities/assignments/assignment_2.ipynb
Comment (rachfern) [reply 02_activities/assignments/assignment_2.ipynb]: @anjali-deshpande-hub  is it good now?

**Comment by anjali-deshpande-hub** [comment]
The last commit you made to revert assignment_1.ipynb didn't quite do it. Check https://github.com/rachfern/python/blob/assignment-2/02_activities/assignments/assignment_1.ipynb. It still has changes. 

Another way to check it is to check 'Files Changed' tab in the PR.

File: 02_activities/assignments/assignment_2.ipynb
Comment (anjali-deshpande-hub) [reply 02_activities/assignments/assignment_2.ipynb]: This problem is fixed.

File: 02_activities/assignments/assignment_1.ipynb
Comment (anjali-deshpande-hub) [inline 02_activities/assignments/assignment_1.ipynb:1]: There are still changes to assignment_1.ipynb in this PR

**Comment by anjali-deshpande-hub** [comment]
Please don't merge the code to main until the PR is approved. One issue is still pending with it.

**Comment by anjali-deshpande-hub** [comment]
Since you have merged your code to 'main' branch, your main branch has changes to assignment_1.ipynb file in it. When you do `git checkout main -- assignment_1.ipynb`, you get the changed version of that file. Therefore, its not working. 
I will mark the PR as approved. 

For future, please don't close the PR/merge code until the PR is reviewed(or re-reviewed whatever the case may be) and tagged as 'Approved'. Thanks!

**Pull request merged**
